data/json/requirements/materials.json

```diff
@@ -125,109 +144,139 @@
     "components": [
       [
         [ "rag", 1 ],
+        [ "cotton_patchwork", 8 ],
         [ "sheet_cotton_patchwork", 1 ],
         [ "sheet_cotton", 1 ],
-        [ "lycra_patch", 1 ],
+        [ "lycra_patch", 8 ],
+        [ "sheet_lycra_patchwork", 1 ],
         [ "sheet_lycra", 1 ],
-        [ "nylon", 1 ],
+        [ "nylon", 8 ],
         [ "sheet_nylon_patchwork", 1 ],
         [ "sheet_nylon", 1 ],
-        [ "nomex", 1 ],
+        [ "nomex", 8 ],
         [ "sheet_nomex_patchwork", 1 ],
         [ "sheet_nomex", 1 ]
       ]
     ]
   },
+  {
+    "id": "fabric_standard_permeable_small",
+    "type": "requirement",
+    "//": "Materials used for crafting or repairing woven fabrics",
+    "components": [ [ [ "rag", 1 ], [ "cotton_patchwork", 1 ], [ "lycra_patch", 8 ], [ "nylon", 8 ], [ "nomex", 8 ] ] ]
+  },
   {
     "id": "fabric_standard_nostretch",
     "type": "requirement",
     "//": "Materials used for crafting or repairing non-form fitting clothing",
     "components": [
       [
         [ "rag", 1 ],
+        [ "cotton_patchwork", 8 ],
         [ "sheet_cotton_patchwork", 1 ],
         [ "sheet_cotton", 1 ],
-        [ "nylon", 1 ],
+        [ "nylon", 8 ],
         [ "sheet_nylon_patchwork", 1 ],
         [ "sheet_nylon", 1 ],
-        [ "nomex", 1 ],
+        [ "nomex", 8 ],
         [ "sheet_nomex_patchwork", 1 ],
         [ "sheet_nomex", 1 ]
       ]
     ]
   },
+  {
+    "id": "fabric_standard_nostretch_small",
+    "type": "requirement",
+    "//": "Materials used for crafting or repairing non-form fitting clothing",
+    "components": [ [ [ "rag", 1 ], [ "cotton_patchwork", 1 ], [ "nylon", 1 ], [ "nomex", 1 ] ] ]
```

Comment:
I should have made this a review sorry so it only pinged once, sorry.

Reply:
We discussed this in discord and this requirement should not exist (or at least not be used) until items can inherit stats/materials from the items they're crafted from.

Reply:
As for the rag references, I’ll hit them later when I’m on PC. Somehow notepad++ search missed them.

Reply:
Double checked the fabric_standard_nostretch_small. it's not used in any recipes now, but will remain for future usage when we *do* have the ability to inherit stats based on materials when crafting.
fabric_standard_nostretch *is* used, but often times it's obvious that you're just using a non-stretchable fabric as a backing for armor or a something like a connecting or flex panel. I did find a couple that used it incorrectly still and moved them appropriately. Specifically, I know that it's used in some of the armor suits - but it's armor that's made of other pieces of armor (I.E. Survivor Duster), where the fabric would ideally be between the layers. We can modify it if it turns out to be a problem, though.

> Ok I've spent the morning reviewing this to help get this completed and merged before changes seep in. It's internally consistent and and should work well. Thank you for all the work this has taken.

No problem! Thanks for taking the time to review it. There's going to be some rebalancing needed for sure, but this feels good to complete, as the current system was a MESS.

I'm going to just mark the rest of these as resolved as soon as i'm done linting and push this commit addressing these comments.